### ISSUE TITLE
FullCalendar: Deprecated component

### ISSUE THREAD

**Issue by tugcekucukoglu** [primefaces/primevue#3827]

**Comment by simon-davey** [comment]
Hi @tugcekucukoglu,

Can you give us some more information on this breaking change, please. Specifically, why is this component being deprecated and will there be an alternative calendar UI (fully featured rather than the date-picker calendar) added in 3.27?

**Comment by simon-davey** [comment]
Would it be possible to mark the component as deprecated but not actually remove it until the PrimeVue calendar is released? 
Removing it before This would give users of the component an opportunity to move t

**Comment by ineshbose** [comment]
I've upgraded to 3.27 as well but as using FullCalendar in my app, so I need to downgrade - I know a replacement is planned for Q3, but till then the component should be deprecated and not actually removed (both are different!)

**Comment by arnyee** [comment]
Can you take back Fullcalendar or please show u what is the new actual  component

**Comment by Ancient-Dragon** [comment]
> Would it be possible to mark the component as deprecated but not actually remove it until the PrimeVue calendar is released? Removing it before the PrimeVue calendar is ready will block users from upgrading to 3.27

I agree with this, it should be marked as deprecated but still usable to allow people the time to migrate off rather than blocking people from upgrading.

**Comment by Peter-J-Brown** [comment]
I'd like the echo the comments above, a transition period between deprecation and outright removal would have been appreciated here

**Comment by arnyee** [comment]
@tugcekucukoglu Please can you give us a solution here? What we need to use?

**Comment by ineshbose** [comment]
I copied contents from https://github.com/primefaces/primevue/blob/ac87f9a5d568f9ae413879a87a7bed6417a2f1fa/components/lib/fullcalendar/FullCalendar.vue and added it to my app as a workaround.

**Comment by arnyee** [comment]
Very nice solution @ineshbose Thx mate

**Comment by joaopaulocruz** [comment]
Just change 

import FullCalendar from "primevue/fullcalendar"

to 

import FullCalendar from "@fullcalendar/vue3"
